rama-cli/Cargo.toml

```diff
@@ -25,14 +25,15 @@ itertools = { workspace = true }
 mimalloc = { workspace = true, optional = true }
 opentelemetry-otlp = { workspace = true }
 rama = { version = "0.3.0-alpha.2", path = "..", features = [
-    "compression",
-    "boring",
-    "cli",
-    "tcp",
-    "udp",
-    "http-full",
-    "proxy-full",
-    "opentelemetry",
+  "compression",
+  "boring",
+  "cli",
+  "tcp",
+  "udp",
+  "http-full",
+  "proxy-full",
+  "opentelemetry",
+  "rustls",
```

Comment:
there should be no need for introducing rustls here, you should be able to do it with boring